build.sbt

```diff
@@ -28,3 +27,6 @@ lazy val `scala-library-next` = crossProject(JVMPlatform, JSPlatform)
   .jsSettings(
     Test / fork := false,
   )
+
+lazy val scalaLibraryNextJVM = scalaLibraryNext.jvm
+lazy val scalaLibraryNextJS  = scalaLibraryNext.js
```

Comment:
I'm hoping these would help. Do they work in practice?

Reply:
Yes. I frequently see this done "in the wild" (in repos included in the community build).

Reply:
This was necessary before sbt 1.2.0, and so it's likely that you see a lot of projects still using it. But in fact it's completely redundant now.

Reply:
@sjrd given sbt/sbt#4210, it doesn't seem redundant to me. if I remove this, then I can e.g. `show scalaLibraryNextJVM/scalaVersion` but I cannot e.g. `set scalaLibraryNextJVM/scalaVersion := "2.13.2"`, I must instead `set scalaLibraryNext.jvm/scalaVersion := "2.13.2"`. Do you understand that this mismatch between `show` and `set` that I'm seeking to avoid? (And am I still missing something?)

Reply:
> Do you understand that this mismatch between show and set is what I'm seeking to avoid?

I had not understood that before. Given that, my comment is moot, indeed.

Reply:
> This was necessary before sbt 1.2.0, and so it's likely that you see a lot of projects still using it. But in fact it's completely redundant now.

Wait, what changed in sbt 1.2 to make this redundant?

Reply:
The introduction of `CompositeProject`, which is then leveraged by `sbt-crossproject` by extending it in `CrossProject`.

Reply:
Ah, right. TY